tests/e2e.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+gcloud auth activate-service-account --key-file=/secret/gcp-credentials/key.json
+# gcloud builds submit --config cloudbuild.yaml .
```

Comment:
where are we running the integration tests? I only see unit tests are bing run in unit.sh

Reply:
I disabled the integration tests because I wasn't able to get them running successfully. The infrastructure does run tests successfully, but theres probably a difference in environment/bug in the tests that hangs.